### PR TITLE
DDCNL-1864 

### PR DESCRIPTION
**Pull request by oc-ste**

### CODE REVIEW

File: test/viewModels/HomeViewModelSpec.scala [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+package viewModels
+
+import models.{ActivatedOnlineFilerSelfAssessmentUser, NonFilerSelfAssessmentUser, NotEnrolledSelfAssessmentUser, NotYetActivatedOnlineFilerSelfAssessmentUser, WrongCredentialsSelfAssessmentUser}
+import uk.gov.hmrc.domain.SaUtr
+import util.BaseSpec
+import viewmodels.HomeViewModel
+
+class HomeViewModelSpec extends BaseSpec {
+
+  val utr = "1111111111"
```
Comment (shikkhasingh) [inline test/viewModels/HomeViewModelSpec.scala:26]: Can we please use SaUtrGenerator here?

Comment (oc-ste) [reply test/viewModels/HomeViewModelSpec.scala]: Genned UTRs in all tests

File: test/views/html/homeSpec.scala [hunk 2]
```diff
@@ -17,109 +17,82 @@
...
+      implicit val userRequest = buildUserRequest(request = FakeRequest())
 
+      val view = views.html.home(homeViewModel).toString
+
+      view should not contain messages("label.home_page.utr")
+    }
+
+    "should show the UTR if the user is a self assessment user" in {
+      implicit val userRequest = buildUserRequest(request = FakeRequest())
+      val utr = "123456789"
```
Comment (shikkhasingh) [inline test/views/html/homeSpec.scala:91]: Same here

Comment (oc-ste) [reply test/views/html/homeSpec.scala]: done

**Pull request merged**